translate/filters/test_checks.py

```diff
@@ -1374,3 +1382,33 @@ def test_skip_checks_per_language_in_some_checkers():
     failures = stdchecker.run_filters(unit)
     # But it is not in StandardChecker.
     assert 'accelerators' in failures.keys()
+
+
+def test_skip_checks_for_l20n_complex_units():
+    """Test some checks are skipped for some languages in Mozilla checker."""
+    from translate.storage import base
+
+    str1, str2, __ = strprep(
+        u"""{ PLURAL($num) ->
+          [one] { $num } fish.
+         *[other] { $num } fishes.
+        }""",
+        u"""{ PLURAL($num) ->
+         *[other] { $num } balık.
+        }"""
+    )
+    unit = base.TranslationUnit(str1)
+    unit.target = str2
+
+    l20nchecker = checks.L20nChecker()
+    failures = l20nchecker.run_filters(unit)
+    assert 'variables' not in failures.keys()
+    assert 'brackets' not in failures.keys()
+    assert 'newlines' not in failures.keys()
```

Comment:
I'm not sure I follow.
`test_variables_l20n` work with `variables` check. And the rest of checks are the same as in `MozillaChecker`.